### ISSUE TITLE
[Feature] Re-run test from inspector

### ISSUE THREAD

**Issue by aisabel** [microsoft/playwright#13345]
Once we have enabled inspector mode, run the test either "step by step", or by clicking "play" button only once.
Would be nice to have the option to re-run this test without having to close inspector and open again.

**Comment by aisabel** [comment]
Alright thank you for your response :)
